interview-questions.md

```diff
@@ -7,44 +7,97 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: 
+  Start by adding belongs_to under the model Students file. 
+  You would want to create a new timestamped migrations file. 
+  Edit new migration file adding a column for the foreign key (Cohortid) set to recieve integers.
+  rails db:migrate to move the new migrations file to the schema.
+  I believe another step is involved but without working it out I don't know what I am missing.
 
   Researched answer:
-
+  create model student if it hasn't been done yet
+  add column cohort_id accepting integers has to be added to the model
+   in  both models we need to modive them with belongs_to :cohort and has_many :students
+foreign key would be in the belongs to model so students in this case
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: Oh god, good question. 
```

Comment:
Yes, it is a good question and a likely one that will be asked in an interview.  Would recommend reviewing the [RESTful routes](https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/rails/restful-routes-crud.md) section in the syllabus.